cpp/src_prims/label/classlabels.cuh

```diff
@@ -153,7 +153,8 @@ void make_monotonic(Type *out, Type *in, size_t N, cudaStream_t stream,
   dim3 blocks(ceildiv(N, TPB_X));
   dim3 threads(TPB_X);
 
-  std::shared_ptr<deviceAllocator> allocator(new defaultDeviceAllocator);
+  std::shared_ptr<deviceAllocator> allocator(
```

Comment:
We should have had this prim accept an `allocator` object in its arg-list, instead. Can you make this change, please?